kalite/contentload/management/commands/generate_assessment_zips.py

```diff
@@ -33,7 +35,7 @@ def handle_noargs(self, **options):
         # without redownloading all files. Not possible currently because ZipFile has no `delete`.
         logging.info("downloading images")
         with open(ZIP_FILE_PATH, "w") as f:
-            zf = zipfile.ZipFile(ZIP_FILE_PATH, "w")  # zipfile.ZipFile isn't a context manager yet for python 2.6
+            zf = zipfile.ZipFile(f, "w")  # zipfile.ZipFile isn't a context manager yet for python 2.6
```

Comment:
Just cleaning up a code smell I did here, which is not use the `f` I just opened.